src/woo_publications/publications/admin.py

```diff
@@ -386,7 +391,10 @@ class PublicationAdmin(AdminAuditLogMixin, admin.ModelAdmin):
         ("archiefactiedatum", PastAndFutureDateFieldFilter),
     )
     date_hierarchy = "registratiedatum"
-    inlines = (DocumentInlineAdmin,)
+    inlines = (
+        DocumentInlineAdmin,
+        PublicationIdentifierInlineAdmin,
+    )
```

Comment:
```suggestion
    inlines = (
        PublicationIdentifierInlineAdmin,
        DocumentInlineAdmin,
    )
```
let's swap them around because the documents are quite bulky and it's a lot of scrolling to get to the identifiers